fcrepo4/fcrepo4
# refactor event api, removing info map

# What does this Pull Request do?

Refactors Event, replacing the info map with fields and removing OptionalValues.

# What's new?

Nothing new.

# How should this be tested?

Mutate a resource and [see events](https://wiki.lyrasis.org/display/FEDORA51/How+to+Inspect+Event+Messages+Generated+by+Fedora).

# Interested parties

@fcrepo4/committers
